ivc/src/ivc/columns.rs

```diff
@@ -212,6 +218,9 @@ pub type IVCPoseidonColumn = PoseidonColumn<IVC_POSEIDON_STATE_SIZE, IVC_POSEIDO
 // TODO: Can we pass just one coordinate and sign (x, sign) instead of (x,y) for hashing?
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub enum IVCColumn {
+    /// Selector for blocks. Inner usize is ∈ [0,#blocks).
```

Comment:
I cannot find the up-to-date description of each different block. Is it the ones at the top of file?
My question is related to the function `build_selectors`, to verify the selectors are populated according to the spec.

Reply:
Yes, the general description of each block is in the top of `columns.rs`. I don't know how to create `const usize` definitions for block sizes, because they depend on `N_COL_TOTAL` and `N_CHALS` and generic consts in rust are not supported....

Reply:
I fixed this by adding `block_height` function... but I see it's merged? heh